boards/amd/versalnet_rpu/doc/index.rst

```diff
@@ -0,0 +1,79 @@
+.. zephyr:board:: versalnet_rpu
+
+Overview
+********
+This configuration provides support for the RPU(R52), real-time processing unit on Xilinx
+Versal Net SOC, it can operate as following:
+
+* Two independent R52 cores with their own TCMs (tightly coupled memories)
+* Or as a single dual lock step unit with the TCM.
+
+This processing unit is based on an ARM Cortex-R52 CPU, it also enables the following devices:
+
+* ARM GIC v3 Interrupt Controller
+* Global Timer Counter
+* SBSA UART
+
+Hardware
+********
+Supported Features
+==================
+
+The following hardware features are supported:
```

Comment:
Think this is replaced with new command @kartben

Reply:
could you please provide a reference example